rest-api/src/main/java/gov/cms/qpp/conversion/api/services/StorageServiceImpl.java

```diff
@@ -57,7 +60,7 @@
 		ObjectMetadata s3ObjectMetadata = new ObjectMetadata();
 		s3ObjectMetadata.setContentLength(size);
 
-		PutObjectRequest putObjectRequest = new PutObjectRequest(bucketName, keyName, inStream, s3ObjectMetadata)
+		PutObjectRequest putObjectRequest = new PutObjectRequest(bucketName, keyName, inStream.get(), s3ObjectMetadata)
```

Comment:
Does this work after retries?  I'm thinking not since the `InputStream` is gotten right here, but the method that is retried is below (`asynchronousAction` method) with the `PutObjectRequest` parameter coming from this variable.

Reply:
fixed